independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -1,5 +1,9 @@
 package io.quarkus.devtools.commands.handlers;
 
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartData.QuarkusDataKey.PROJECT_PARENT_ARTIFACT_ID;
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartData.QuarkusDataKey.PROJECT_PARENT_GROUP_ID;
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartData.QuarkusDataKey.PROJECT_PARENT_PATH;
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartData.QuarkusDataKey.PROJECT_PARENT_VERSION;
```

Comment:
@ia3andy for pasts PR I understood this was the right place for such keys, after CLI 2.0, it seems that everything is going to ProjectGenerator. Should I move these too?

Reply:
Keep it like that, I will give it a big clean at some point :)	

`ProjectGenerator` is meant to be deleted. I am not sure yet what's the best solution because I don't want to many layers and still Codestarts keys and Commands keys can be slightly different...

cc @ebullient

Reply:
The cross-package includes are kind of a mess, yes.. I doubt ProjectGenerator (as a place for project creation constants) will actually end up being removed.. especially as the middle layer command layer is getting removed in some cases.. we'll have to walk through it (making a map to make a map to unpack it again later just adds time.. )

Reply:
@ebullient if we keep one it will be `QuarkusCodestartData` because this is what's used by the codestarts below. `ProjectGenerator` is a set of constant that were used in the legacy code generator and have been kept for backward compatibility which isn't necessary anymore (they are converted to QuarkusCodestartData):
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartData.java#L110